HOWTO_RELEASE.md

```diff
@@ -0,0 +1,106 @@
+# Steps
+
+Release early, release often. Don't be lazy.
+
+To use this doc: just replace 0.1.0 with the major.minor.patch version of
+the release. The sequence of commands below should be good to copy and
+paste, but do please pay attention to details!
```

Comment:
I believe this was meant to be "please do" instead of "do please"?

Reply:
Just removed the `do`. And also replaced the numbers for generic X.Y.Z (was a leftover).